Add XML Web Services to 21.0.0.2-beta blog (#1057)
OpenLiberty/blogs: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc

Additional information about the Jakarta Security 2.0 specification can be found link:https://jakarta.ee/specifications/security/2.0/[here].

[#xmlws]
=== Jakarta XML Web Services 3.0

> Jakarta XML Web Services 3.0 is an API that enables applications to create SOAP based Web Services. Support provides the following updated APIs: XML Web Services 3.0, Jakarta Web Services Metadata 3.0, Jakarta SOAP with Attachments 2.0. Support also introduces the `xmlWS-3.0` feature, as well as the Application Client Container support for XML Web Services 3.0 via the `jakartaeeClient-9.0` convenience feature.

Reviewer: ```suggestion
Jakarta XML Web Services 3.0 is an API that enables applications to create SOAP based Web Services. The `xmlWS-3.0` feature provides API updates to the Jakarta XML Web Services 3.0 API, Jakarta Web Services Metadata 3.0 API and Jakarta SOAP with Attachments 2.0 API. The `jakartaeeClient-9.0` feature has been also been updated to provide Application Client Container support for the Jakarta XML Web Services 3.0 API.
```

Author: Updated paragraph.

The `xmlWS-3.0` feature provides the Jakarta EE 9 XML Web Services 3.0 tools for Top Down and Bottom Up Web Services Creation: `wsgen` and `wsimport`. These can be found in a new location in the Open Liberty image: wlp/bin/xmlWS/. Both commands preform nearly identical to the Jakarta EE 8 tooling, with the exception that the `-target` argument is no longer required.

> The implementation of Jakarta XML Web Services 3.0 in Open Liberty is based on Apache CXF 3.4.1. This provides numerous improvements and fixes over the forked version of CXF 2.6.2 in use by the Jakarta EE 8 `jaxws-2.3` feature (Note: Not all CXF 3.4.1 functionality is available in `xmlWS-3.0`, but watch this space for updates.)

Reviewer: I think maybe this paragraph should be removed... I don't like advertising a specific implementation version or hinting at support for future function.

Author: Removed paragraph and changed `preform` -> `perform`

To enable the new Jakarta EE 9 XML Web Services 3.0 feature, add the xmlWS-3.0 feature to your server.xml. Here's an example of what the feature configuration looks like:

[source, xml]
----
 <featureManager>
       <feature>xmlWS-3.0</feature>
 </featureManager>
----

To run Jakarta XML Web Services on the Application Client Container: Here's an example of what the client.xml configuration looks:

[source, xml]
----
 <featureManager>
       <feature>jakartaeeClient-9.0</feature>
 </featureManager>
----

> For more information see the link:https://jakarta.ee/specifications/xml-web-services/[Jakarta XML Web Services Specification], and the link:https://cxf.apache.org/docs/34-migration-guide.html[Apache CXF 3.4 Migration Guide].

Reviewer: Link directly to the 3.0 version: https://jakarta.ee/specifications/xml-web-services/3.0/

Reviewer: I think the Apache CXF Migration Guide link should be removed

Author: Updated link and remove link to migration guide.
